modules/ROOT/pages/environment-limitations.adoc

```diff
@@ -0,0 +1,41 @@
+= Limitations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The following limitations exist for Runtime Fabric. These are important to understand when installing, configuring, 
```

Comment:
Minor, but from a marketing point of view, I wonder if we can make this more positive.

"Runtime Fabric has the following limitations.". -- insure if there's room for something better without sacrificing quality.

Reply:
Re-worded this - thank you!